src/adafruit_blinka/microcontroller/rockchip/rk3566/pin.py

```diff
@@ -242,6 +242,16 @@
         globals()[alias + "_RX"] = GPIO2_A3
         uartPorts.append((int(alias[-1]), GPIO2_A4, GPIO2_A3))
 
+if board in ("RADXA_ZERO3"):
```

Comment:
A bit late to the party, but I believe this might be wrong.
Here the `()` have no effect (they don't create a tuple), so this is a substring check.  If `board` is equal to `"RADXA_ZERO3"` it will still work, but it will also work if `board` is e.g. `"RADXA"` or `"ZERO"`:
```py
>>> "RADXA_ZERO3" in ("RADXA_ZERO3")
True
>>> "RADXA" in ("RADXA_ZERO3")
True
>>> "ZERO" in ("RADXA_ZERO3")
True
```

You probably want `if board in {"RADXA_ZERO3"}:` (using sets) or `if board in ("RADXA_ZERO3",):` (using tuples).

Note that the same pattern is also found above, at line 209.

@makermelissa, let me know what you think and if you would like me to make a PR to fix this.